config/v3/account_closure/send.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Account Closure (request)  Required scope(s): [admin]",
+    "description": "This call allows clients to close all their accounts (including virtual-money account). It is assumed that all their accounts (excluding virtual) have no balance left.",
+    "hidden": 1,
```

Comment:
Calls having this key shouldn't end up here. Maybe need to use the script, not manually copying them.

Reply:
fixed